flink-libraries/flink-ml/src/main/scala/org/apache/flink/ml/recommendation/ALS.scala

```diff
@@ -273,6 +308,14 @@ object ALS {
     val defaultValue: Option[Int] = Some(10)
   }
 
+  case object ImplicitPrefs extends Parameter[Boolean] {
```

Comment:
I suggest to leave it at 10 and do not give any advise to use highest possible number. It should work out-of-the-box, and to get good results, the user must fit the parameters to the use-case (such as the lambda coefficient of the regularization). Btw. Spark uses 10 by default.